.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Code Question
+about: Ask a question about the code
+title: ''
+labels: 'Type: Question'
+assignees: ''
+
+---
+
+<!--
+Thank you for taking an interest in the OpenZFS codebase.
+
+Please be aware that most questions are preferably asked in the mailinglist first.
```

Comment:
mailinglist -> mailing list

Reply:
Yeah sorry, in dutch we combine just about everything...

Reply:
I thought that might be the case.